tests/test_sigpig_join_notification_api.py

```
@@ -0,0 +1,206 @@
+    api_client,
+    build_headers,
+    create_user,
+    db_session,
+    monkeypatch,
+):
+    """문자 발송 실패가 발생해도 가입 자체는 성공으로 유지되는지 확인한다."""
+    owner, _ = create_user()
+    applicant, applicant_token = create_user()
+    sig = _create_sig_for_join(db_session, owner_id=owner.id)
+
+    async def failing_send_sms(*, to: str, content: str):
```

Ruff (0.15.2)

[warning] Unused function argument: `to`

(ARG001)